draft-ietf-httpbis-http2bis.xml

```diff
@@ -2736,8 +2736,17 @@ CONTINUATION Frame {
         <section>
           <name>Field Validity</name>
           <t>
-            HPACK is capable of carrying field names or values that are not valid in HTTP.  Though
-            HPACK can carry any octet, fields are not valid in the following cases:
+            HPACK is capable of carrying field names or values that are not valid in HTTP.
+          </t>
+          <t>
+            <xref target="HTTP" section="5.1"/> and <xref target="HTTP" section="5.5"/> define field
+            names and values and what characters are valid.  A recipient MAY treat a message that
```

Comment:
I don't think that `MAY` does enough to satisfy #902.
I think this paragraph should be the first requirement in the list below and say that in H2 a field MUST be treated as malformed if it is invalid in HTTP - unless it is a pseudo header.  Specifically we should say:

HPACK can carry non valid fileds.
All H2 impls MUST validate as follows:
 + must be a valid HTTP field or a psuedo field
 + names must be lower case
 + must not contain zero value etc.
 + must not start or end with whitespace